test/conftest.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+# SPDX-FileCopyrightText: : 2017-2024 The PyPSA-Eur Authors
+#
+# SPDX-License-Identifier: MIT
+
+# coding: utf-8
+
+import pathlib
+import shutil
+
+import pypsa
+import pytest
+import yaml
+
+
+@pytest.fixture(scope="function")
+def get_power_network_scigrid_de():
```

Comment:
```suggestion
def scigrid_network():
```
I am more a fan of simpler fixture names. But I guess that's taste

Also `ac_dc_network` and just `config`